internal/sirius/edit_dates_test.go

```diff
@@ -0,0 +1,94 @@
+package sirius
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/pact-foundation/pact-go/dsl"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestEditDates(t *testing.T) {
+	pact := newPact()
+	defer pact.Teardown()
+
+	testCases := []struct {
+		name          string
+		setup         func()
+		cookies       []*http.Cookie
+		expectedError func(int) error
+	}{
+		{
+			name: "OK",
+			setup: func() {
+				pact.
+					AddInteraction().
+					Given("I have a pending case assigned").
+					UponReceiving("A request to edit the dates").
+					WithRequest(dsl.Request{
+						Method: http.MethodPut,
+						Path:   dsl.String("/api/v1/lpas/800"),
+						Headers: dsl.MapMatcher{
+							"X-XSRF-TOKEN":        dsl.String("abcde"),
+							"Cookie":              dsl.String("XSRF-TOKEN=abcde; Other=other"),
+							"OPG-Bypass-Membrane": dsl.String("1"),
+						},
```

Comment:
Should probably also add the body to ensure that the code is sending the right thing and that it's accepted by the provider.
```go
					Body: map[string]interface{}{
						"rejectedDate": "04/03/2022",
					},
```

Reply:
umm yep, totally missed this